pycbc/workflow/datafind.py

```diff
@@ -751,15 +751,6 @@ def convert_cachelist_to_filelist(datafindcache_list):
                     # Datafind returned a URL valid on the osg as well
                     # so add the additional PFNs to allow OSG access.
                     currFile.add_pfn(frame.url, site='osg')
-                    currFile.add_pfn(frame.url.replace(
-                        'file:///cvmfs/oasis.opensciencegrid.org/',
-                        'root://xrootd-local.unl.edu/user/'), site='osg')
```

Comment:
In other places, the xrootd has been retained (e.g. the changes in `pycbc/workflow/pegasus_files/pegasus-properties.conf`) - should this be the same?

Reply:
This one should go, as the OSG site is a LVK thing and this does not use root. I'm never sure what this was added for, but if non-LVK people still use this, it would be needed in pegasus-properties.conf (and removing that would be out of scope for this PR).